lib/oban/plugins/stager.ex

```diff
@@ -55,18 +74,38 @@ defmodule Oban.Plugins.Stager do
 
   @impl GenServer
   def handle_info(:stage, %State{} = state) do
+    start_metadata = %{config: state.conf, plugin: __MODULE__}
+
+    :telemetry.span(
```

Comment:
Same comment here about the vertical space. It also needs to handle the `{:ok, false}` xact lock return value.